lsp/src/text_document.ml

```diff
@@ -32,8 +32,9 @@ let find_utf8_pos =
     if char = 0 then Uutf.decoder_byte_count dec
     else
       match Uutf.decode dec with
-      | `Malformed _ | `Await -> raise Invalid_utf8
-      | `End -> assert false
+      | `Malformed m -> raise (Invalid_utf m)
```

Comment:
When `Uutf.decode` returns `Malformed s`, what's `s` here? Just bytes read with `Uutf.Manual.src` call?

Reply:
The malformed byte sequence.